src/python/include/Subsampling_interface.h

```diff
@@ -27,23 +28,22 @@ namespace subsampling {
 
 using Subsampling_dynamic_kernel = CGAL::Epick_d< CGAL::Dynamic_dimension_tag >;
 using Subsampling_point_d = Subsampling_dynamic_kernel::Point_d;
-using Subsampling_ft = Subsampling_dynamic_kernel::FT;
 
 // ------ choose_n_farthest_points ------
 std::vector<std::vector<double>> subsampling_n_farthest_points(const std::vector<std::vector<double>>& points,
                                                                unsigned nb_points) {
   std::vector<std::vector<double>> landmarks;
-  Subsampling_dynamic_kernel k;
-  choose_n_farthest_points(k, points, nb_points, random_starting_point, std::back_inserter(landmarks));
+  choose_n_farthest_points(Euclidean_distance(), points, nb_points,
+                           random_starting_point, std::back_inserter(landmarks));
```

Comment:
Did you benchmark this modification ? I don't know if CGAL made something smart in `squared_distance_d_object` compared to our naive `Euclidean_distance()` implementation.

Reply:
I didn't benchmark. CGAL doesn't do anything smart there, except for not computing the square root. Well it also uses `CGAL::square` instead of doing `tmp*tmp`, but at least for `double` that doesn't make any difference.